core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -126,8 +126,7 @@ THE SOFTWARE.
     <script src="${resURL}/scripts/prototype.js" type="text/javascript"/>
     <script src="${resURL}/scripts/behavior.js" type="text/javascript"/>
 
-    <!-- we include our own prototype.js, so don't let stapler pull in another. -->
-    <st:adjunct assumes="org.kohsuke.stapler.framework.prototype.prototype" includes="org.kohsuke.stapler.bind"/>
+    <st:adjunct includes="org.kohsuke.stapler.bind"/>
```

Comment:
There is copypasta of this code in three places:

- https://github.com/jenkinsci/scriptler-plugin/blob/f42c1538652df1af88b2d75be5f42090c7b1329d/src/main/resources/org/jenkinsci/plugins/scriptler/builder/ScriptlerBuilder/config.jelly#L4
- https://github.com/jenkinsci/build-monitor-plugin/blob/312385bf266bae8fcde82990ac2753002a06f86f/build-monitor-plugin/src/main/resources/com/smartcodeltd/jenkinsci/plugins/buildmonitor/BuildMonitorView/index.jelly#L33
- https://github.com/jenkinsci/hp-operations-orchestration-plugin/blob/eb027aa3db9cda0cd1bd493bfafb1dd7e33ab068/src/main/resources/com/hp/mercury/ci/jenkins/plugins/OOBuildStep/config.jelly#L30

When this is released, it will result in a scary-looking but harmless warning (stack trace) being printed to the Jenkins log:

```
2023-04-04 03:12:57.973+0000 [id=264]        WARNING        o.k.s.f.adjunct.AdjunctsInPage#findNeeded: No such adjunct found: org.kohsuke.stapler.framework.prototype.prototype
org.kohsuke.stapler.framework.adjunct.NoSuchAdjunctException: Neither org.kohsuke.stapler.framework.prototype.prototype.css, .js, .html, nor .jelly were found
        at org.kohsuke.stapler.framework.adjunct.Adjunct.<init>(Adjunct.java:125)
        at org.kohsuke.stapler.framework.adjunct.AdjunctManager.get(AdjunctManager.java:148)
        at org.kohsuke.stapler.framework.adjunct.AdjunctsInPage.findNeeded(AdjunctsInPage.java:161)
        at org.kohsuke.stapler.framework.adjunct.AdjunctsInPage.assumeIncluded(AdjunctsInPage.java:131)
        at org.kohsuke.stapler.framework.adjunct.AdjunctsInPage.assumeIncluded(AdjunctsInPage.java:125)
```

I don't think this is a huge deal that warrants a release of those plugins with their baseline bumped up to a weekly. Rather, I plan to wait until this PR makes it into an LTS, and then file PRs bumping those plugins to that LTS release and fixing the copypasta so that it matches the change made in core, which will remove the scary warning.

Reply:
> Rather, I plan to wait until this PR makes it into an LTS, and then file PRs bumping those plugins to that LTS release and fixing the copypasta so that it matches the change made in core, which will remove the scary warning.

I have followed up and filed PRs against `scriptler-plugin` and `build-monitor-plugin` (the latter of which has been already released). I have not followed up with `hp-operations-orchestration-plugin` because that plugin has not been released in over 7 years and seems unmaintained.